hexa/plugins/connector_iaso/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+
+from . import views
+
+app_name = "connector_iaso"
+
+urlpatterns = [
+    path("<uuid:datasource_id>/", views.datasource_index, name="datasource_index"),
+    path(
+        "<uuid:iasoaccount_id>/form/<int:iaso_id>/",
+        views.iasoform_detail,
+        name="iasoform_detail",
+    ),
```

Comment:
Question here : do we also want a `form_list` view? Or does the account detail view list all the forms without the need for a "view all forms" screen ?